README.md

```diff
@@ -38,10 +38,10 @@ This project use `prettier` and `eslint` to enforce good code formatting practic
 
 ### Prettier
 
-Prettier is installed as a dependency and will run a check automatically before every `git commit`. There are also extensions you can use to run Prettier on save within your IDE. To run prettier manually:
+Prettier is installed as a dependency and will run a check automatically before every `git commit`. There are also extensions you can use to run Prettier on save within your IDE. To run prettier locally, run the following command or see the [Prettier Docs](https://prettier.io/docs/en/index.html):
```

Comment:
It would be helpful to distinguish between the commit-hook that runs prettier _as a check_, and your "run prettier locally" which runs prettier to _edit-in-place_ ("Beware!", as it says in the command-line help).

Something like:

---
[Prettier](https://prettier.io/) is  "an opinionated code formatter." It is installed as a dependancy and will automatically check your code before every `git commit` (a pre-commit hook is installed via `husky`).

You can run Prettier within your IDE, such as automatically when saving. See the [Prettier Docs](https://prettier.io/docs/en/index.html).

To run Prettier manually, which will rewrite your code in its style:

---